apps/admin/frontend/test/helpers/api_mock.ts

```diff
@@ -326,10 +326,8 @@ export function createApiMock(
       );
     },
 
-    expectWriteBallotPackageSignatureFile(ballotPackagePath: string) {
-      apiClient.writeBallotPackageSignatureFile
-        .expectCallWith({ ballotPackagePath })
-        .resolves();
+    expectSaveBallotPackageToUsb(result: Result<void, 'no_usb_drive'> = ok()) {
```

Comment:
Maybe also worth it to give the ballot package save result a defined type for use here and in the backend?

Reply:
Agreed https://github.com/votingworks/vxsuite/pull/3554/commits/18f42ecab9d61ced650358f0f80bb74ff382ac26